Cannot find module or its corresponding type declarations.

```console
error TS2307: Cannot find module '@brillout/json-serializer/parse' or its corresponding type declarations.

error TS2307: Cannot find module '@brillout/json-serializer/stringify' or its corresponding type declarations.
```

Related: https://github.com/vikejs/vike/pull/1955#issuecomment-2462074898

--- a/package.json
+++ b/package.json
@@ -7,13 +7,21 @@
     ".": "./index.mjs",
     "./parse": {
       "worker": "./dist/cjs/parse.js",
-      "node": "./dist/cjs/parse.js",
-      "browser": "./dist/esm/parse.js"
+      "node": {
+        "require": "./dist/cjs/parse.js",
+        "import": "./dist/esm/parse.js"
+      },
+      "browser": "./dist/esm/parse.js",
+      "types": "./dist/esm/parse.d.ts"
     },
     "./stringify": {
       "worker": "./dist/cjs/stringify.js",
-      "node": "./dist/cjs/stringify.js",
-      "browser": "./dist/esm/stringify.js"
+      "node": {
+        "require": "./dist/cjs/stringify.js",
+        "import": "./dist/esm/stringify.js"
+      },
+      "browser": "./dist/esm/stringify.js",
+      "types": "./dist/esm/stringify.d.ts"
     }
   },
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -6,22 +6,18 @@
   "exports": {
     ".": "./index.mjs",
     "./parse": {
-      "worker": "./dist/cjs/parse.js",
       "node": {
-        "require": "./dist/cjs/parse.js",
-        "import": "./dist/esm/parse.js"
+        "require": "./dist/cjs/parse.js"
       },
-      "browser": "./dist/esm/parse.js",
-      "types": "./dist/esm/parse.d.ts"
+      "types": "./dist/esm/parse.d.ts",
+      "default": "./dist/esm/parse.js"
     },
     "./stringify": {
-      "worker": "./dist/cjs/stringify.js",
       "node": {
-        "require": "./dist/cjs/stringify.js",
-        "import": "./dist/esm/stringify.js"
+        "require": "./dist/cjs/stringify.js"
       },
-      "browser": "./dist/esm/stringify.js",
-      "types": "./dist/esm/stringify.d.ts"
+      "types": "./dist/esm/stringify.d.ts",
+      "default": "./dist/esm/stringify.js"
     }
   },
   "scripts": {

